.travis.yml

```diff
@@ -89,10 +89,10 @@ matrix:
 #
 # Misc
 #
-    - <<: *_go
```

Comment:
We had better to remove _go section?

Reply:
https://github.com/osrg/gobgp/pull/1550

Reply:
Thank you for your reviewing. Yes, I agree to remove `_go` section!